django/contrib/admin/templatetags/admin_list.py

```diff
@@ -179,7 +179,7 @@ def make_qs_param(t, n):
 
 def _boolean_icon(field_val):
     icon_url = static(
-        "admin/img/icon-%s.svg" % {True: "yes", False: "no", None: "unknown"}[field_val]
+        "admin/img/icon-%s.svg" % {True: "yes", False: "no"}.get(field_val, "unknown")
```

Comment:
This change is untested and looks unnecessary. `BooleanField` cannot have value other than `False`, `True`, or `None`.

Reply:
This is a leftover from the first version of the testcase.